### PR TITLE
[PL] translation of escript lesson

### PR DESCRIPTION
**Pull request by Koziolek**
In scope of issue #202

### CODE REVIEW

**Comment by doomspork** [comment]
Could you review this PR @kdebowski?

**Comment by nscyclone** [comment]
You may use automatic TOC generation as was described [here](https://github.com/doomspork/elixir-school/pull/352#issuecomment-218850807).

File: pl/lessons/advanced/escripts.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+---
+
+Do stworzenia pliku wykonywalnego w Elixirze służy escript. Escript generuje plik wykonywalny, który może zostać 
+uruchomiony na każdym komputerze, na którym zainstalowano Erlanga.
+
+{% include toc.html %}
+
+## Na początek
+
+By utworzyć plik, który można uruchomić, za pomocą escriptu musimy zrobić tylko drobnych rzeczy: zaimplementować
```
Comment (kdebowski) [inline pl/lessons/advanced/escripts.md]: "zrobić tylko drobnych" -> "zrobić tylko kilka drobnych"

File: pl/lessons/advanced/escripts.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  def escript do
+    [main_module: ExampleApp.CLI]
+  end
+end
+```
+
+## Parsowanie argumentów
+
+Do naszej aplikacji możemy przekazać pewne argumenty z linii poleceń. By je sparsować, użyjemy Elixirowego modułu  
+`OptionParser.parse/2` z opcją `:switches`, która zawiera informacje, iż nasz flaga jest typu logicznego:
```
Comment (kdebowski) [inline pl/lessons/advanced/escripts.md]: nasz -> nasza

File: pl/lessons/advanced/escripts.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+## Tworzenie plików wykonywalnych
+
+Jak już skonfigurujemy aplikację by używała escript, stworzenie pliku wykonywalnego z mixem jest banalne:
+
+```elixir
+$ mix escript.build
+```
+
+Let's take it for a spin:
```
Comment (kdebowski) [inline pl/lessons/advanced/escripts.md]: English text

File: pl/lessons/advanced/escripts.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+```elixir
+$ ./example_app --upcase Hello
+WORLD
+
+$ ./example_app Hi
+Hi
+```
+
```
Comment (kdebowski) [inline pl/lessons/advanced/escripts.md]: There is lack of: "That’s it. We’ve built our first executable in Elixir using escript." translation at the end.

**Comment by kdebowski** [comment]
@Koziolek reviewed. Just few things to fix.

**Pull request merged**